opm/material/densead/Evaluation.hpp

```diff
@@ -255,9 +268,9 @@ class Evaluation
     Evaluation operator-() const
     {
         Evaluation result;
-        result.value_ = -this->value_;
-        for (unsigned varIdx = 0; varIdx < size; ++varIdx)
-            result.derivatives_[varIdx] = - this->derivatives_[varIdx];
+        // set value and derivatives to negative
```

Comment:
*arg*, both (and in particular the missing `operator==()`) sound awfully like a gap in `Dune::FieldVector`. anyway, thanks for the info.